source/contracts/Augur.sol

```diff
@@ -34,6 +35,33 @@ contract Augur is Controlled, Extractable {
     event TokensBurned(address indexed universe, address indexed token, address indexed target, uint256 amount);
     event FeeWindowCreated(address indexed universe, address feeWindow, uint256 startTime, uint256 endTime, uint256 id);
 
+    mapping(address => bool) private universes;
+
+    //
+    // Universe
+    //
+
+    function createGenesisUniverse() public returns (IUniverse) {
+        return createUniverse(IUniverse(0), bytes32(0));
+    }
+
+    function createChildUniverse(bytes32 _parentPayoutDistributionHash) public returns (IUniverse) {
+        require(universes[msg.sender]);
+        IUniverse _parentUniverse = IUniverse(msg.sender);
+        return createUniverse(_parentUniverse, _parentPayoutDistributionHash);
+    }
+
+    function createUniverse(IUniverse _parentUniverse, bytes32 _parentPayoutDistributionHash) private returns (IUniverse) {
+        UniverseFactory _universeFactory = UniverseFactory(controller.lookup("UniverseFactory"));
+        IUniverse _newUniverse = _universeFactory.createUniverse(controller, _parentUniverse, _parentPayoutDistributionHash);
+        universes[_newUniverse] = true;
+        return _newUniverse;
+    }
+
+    function isValidUniverse(IUniverse _universe) public view returns (bool) {
```

Comment:
Consider renaming this to `isKnownUniverse`?  `valid` _feels_ like the wrong word to me here as a universe can be valid but not blessed.  Maybe `Blessed` is better?

Reply:
`Known` sounds good to me